proposals/lip-0046.md

```diff
@@ -242,26 +242,32 @@ The Random module exposes the following function.
 
 This function has exactly the same logic, inputs and outputs as the `isSeedRevealValid` function specified in [the previous section](#isSeedRevealValid).
 
-### Protocol Logic During Block Lifecycle
+### Genesis Block Processing
 
-#### After Genesis Block Execution
+The following step is executed as part of the genesis block processing, see the “Update genesis block schema and processing” LIP for details.
+
+#### Genesis State Initialization
 
 After the genesis block `g` is executed, the following logic is executed:
 
 * Set the `validatorReveals` array to an empty array.
```

Comment:
I would clarify that this array is then stored in the random substore (basically, we need to say that the store is initialized to this default value).

Reply:
Good point, added a point with the substore creation